tests/test_process.py

```diff
@@ -118,8 +118,7 @@ def test_run_cmd_arg_return_code(self):
   def test_run_duplicate_streams_timeout(self):
     """Test raise TimeoutExpired. """
     with self.assertRaises(securesystemslib.process.subprocess.TimeoutExpired):
-      securesystemslib.process.run_duplicate_streams(sys.executable + " --version",
-          timeout=-1)
+      securesystemslib.process.run_duplicate_streams("sleep 3", timeout=-1)
```

Comment:
Although, we currently don't run tests on Windows, I'd like us to keep the tests platform independent. What speaks against running `python -c 'while True: pass'`?